dts/bindings/rtc/microcrystal,rv3028.yaml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2024 ANITRA system s.r.o.
+# SPDX-License-Identifier: Apache-2.0
+
+description: Micro Crystal RV3028 RTC
+
+compatible: "microcrystal,rv3028"
+
+include:
+  - name: rtc-device.yaml
+  - name: i2c-device.yaml
+
+properties:
+  clkout-frequency:
+    type: int
+    enum:
+      - 32768
+      - 8192
+      - 1024
+      - 64
+      - 32
+      - 1
+    description: |
+      Frequency of the CLKOUT signal in Hertz (Hz). When disabled, the CLKOUT pin is LOW.
+
+  backup-switchover:
+    type: string
+    required: true
+    enum:
+      - disabled
+      - direct
+      - level
+    description: |
+      Automatic backup switchover function selection:
+        - disabled: The switchover function is disabled - only one power supply available (VDD)
+        - direct: Direct Switching Mode (DSM): when VDD < VBACKUP, switchover occurs from VDD to
+        VBACKUP without requiring VDD to drop below VTH:LSM (2.0 V)
+        - level: Level Switching Mode (LSM): when VDD < VTH:LSM (2.0 V) AND VBACKUP > VTH:LSM,
+        switchover occurs from VDD to VBACKUP
+
+  aux-voltage-chargeable:
+    type: boolean
+    description: |
+      Enable trickle charging circuit which allows a battery or supercapacitor connected to VBACKUP
+      to be charged.
+
+  trickle-resistor-ohms:
+    type: int
+    default: 3000
+    enum:
+      - 3000
+      - 5000
+      - 9000
+      - 15000
+    description: |
+      Resistance selection for the series current limiting resistor of the trickle charger:
+        - 3000: 3 kOhm (default value on delivery)
```

Comment:
there should be a short sentence saying why 3000 is the default.

Reply:
Thanks for pointing this out. I used `default:` because it matches the vendor's provided default value. Should I make this property required instead? It also depends on `aux-voltage-chargeable` being true.

Reply:
@decsny To avoid incorrect configuration, I've removed the property `aux-voltage-chargeable`, which is redundant. Enabling the trickle charger is done by specifying `trickle-resistor-ohms`.